mesheryctl/internal/cli/root/app/app.go

```diff
@@ -40,9 +40,9 @@ var AppCmd = &cobra.Command{
 	Short: "Cloud Native Apps Management",
 	Long:  `All apps operations: import, list, view, onboard and offboard`,
 	Example: `
-// Base command
-mesheryctl app [subcommand]
-	`,
+Base command:
+  mesheryctl app [subcommand]
+Find more information at: https://docs.meshery.io/reference/mesheryctl#command-reference`,
```

Comment:
```suggestion
Base command:
  mesheryctl app [subcommand]
Find more information at: https://docs.meshery.io/reference/mesheryctl#command-reference`,
```

If this format is used, then the corresponding docs page won't format in proper way. FYI, https://docs.meshery.io/reference/mesheryctl/main 

Suppose if you remove `//` then it'll treat as code block and print in wrong format, similarly the link you've provided will result in code block as well

Reply:
I advise you to set the link elsewhere and NOT in the examples field for a while

Reply:
Is there an example to follow or CLI contributing docs that cover this, @alphaX86?

Reply:
Hi,@alphaX86 @leecalcote, I understand the issue. As per meshery docs, link can print only after the example field. But for cobra-cli, we cannot add reference link below the example field. There is no particular option for that.

Reply:
Familiarizing with this - https://docs.meshery.io/project/contributing/contributing-cli#contributing-to-mesheryctl-documentation - @Althaf66 might shed some new light on the issue.

Reply:
Ok, I've modified the `docs.go` file accordingly to reflect this. `mesheryctl` will show the ref link in the CLI but will stay hidden in the synopsis of command documentation pages.